process/sync/baseSync.go

```diff
@@ -726,6 +731,13 @@ func (boot *baseBootstrap) rollBack(revertUsingForkNonce bool) error {
 			)
 		}
 
+		err = boot.historyRepo.RevertBlock(currHeader, currBody)
+		if err != nil {
+			log.Debug("boot.historyRepo.RevertBlock",
```

Comment:
why isn't this a critical error?

Reply:
I followed the pattern on the previous operation. Should I make this a critical error?

Reply:
I think it will be safer to make it critical error. As this might hide processing/storing data errors from the history db lookup extension mechanism.

Reply:
done.